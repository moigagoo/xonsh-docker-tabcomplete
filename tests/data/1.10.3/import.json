{
        "arguments": [
                "-c", "--change=[]",
                "--help",
                "-m", "--message"
        ],
        "commands": []
}
