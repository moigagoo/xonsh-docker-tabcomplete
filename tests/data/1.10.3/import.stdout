
Usage:	docker import [OPTIONS] file|URL|- [REPOSITORY[:TAG]]

Import the contents from a tarball to create a filesystem image

  -c, --change=[]    Apply Dockerfile instruction to the created image
  --help             Print usage
  -m, --message      Set commit message for imported image
